src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ApproverCaseloadServiceTest.kt

```diff
@@ -41,359 +41,1030 @@ class ApproverCaseloadServiceTest {
   }
 
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Refactored the location of the tests into nested tests so apologies for the reformatting below.